.github/workflows/mobile-deploy-auto.yml

```
@@ -0,0 +1,94 @@
+          else
+            echo "version_bump=build" >> $GITHUB_OUTPUT
+            echo "📈 Version bump: build only"
+          fi
+          
+          # Always deploy both platforms for now (can be enhanced later)
+          echo "platforms=both" >> $GITHUB_OUTPUT
+          echo "should_deploy=true" >> $GITHUB_OUTPUT
+          
+      - name: Log deployment info
+        if: steps.check.outputs.should_deploy == 'true'
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)